templates/projects/directory.html

```diff
@@ -162,397 +144,235 @@ <h1>Open source projects directory</h1>
+                <td>Will Cooke (will.cooke&nbsp;{at}&nbsp;canonical&nbsp;{dot}&nbsp;com)</td>
+              </tr>
+
+              <tr>
+                <th scope='row'>System Settings</th>
+                <td>gsettings-qt,<br /> ubuntu-system-settings-online-accounts,<br /> ubuntu-system-settings,<br /> gsettings-ubuntu-touch-schemas</td>
+                <td>Will Cooke (will.cooke&nbsp;{at}&nbsp;canonical&nbsp;{dot}&nbsp;com)</td>
+              </tr>
+
+              <tr>
+                <th scope='row'>Ubuntu Software Developer Kit (DEPRECATED)</th>
+                <td>ubuntu-ui-extras,<br /> ubuntu-html5-theme,<br /> u1db-qt,<br /> unity-action-api,<br /> qtcreator-plugin-ubuntu,<br /> poppler-qml-plugin,<br /> dee-qt,<br /> ubuntu-settings-components,<br /> ubuntu-ui-toolkit,<br /> cordova-ubuntu,<br /> cordova-cli,<br /> qtcreator-plugin-cmake,<br /> ubuntu-touch-customization-hooks</td>
+                <td>Sergio Schvezov (sergio.schvezov&nbsp;{at}&nbsp;canonical&nbsp;{dot}&nbsp;com)</td>
+              </tr>
+
+              <tr>
+                <th scope='row'>Ubuntu CD image building tools</th>
+                <td>ubuntu-cdimage</td>
+                <td>Steve Langasek (steve.langasek&nbsp;{at}&nbsp;canonical&nbsp;{dot}&nbsp;com)</td>
+              </tr>
+
+              <tr>
+                <th scope='row'>Ubuntu core applications (DEPRECATED)</th>
+                <td>camera-app,<br /> gallery-app,<br /> notes-app,<br /> webbrowser-app,<br /> address-book-app,<br /> messaging-app,<br /> dialer-app,<br /> mediaplayer-app,<br /> ubuntu-keyboard,<br /> clock-app,<br /> calendar-app,<br /> weather-app,<br /> music-app,<br /> calculator-app,<br /> ubuntu-calendar-app</td>
+                <td>Will Cooke (will.cooke&nbsp;{at}&nbsp;canonical&nbsp;{dot}&nbsp;com)</td>
+              </tr>
+
+              <tr>
+                <th scope='row'>Ubuntu crash database</th>
+                <td>whoopsie-daisy</td>
+                <td>Steve Langasek (steve.langasek&nbsp;{at}&nbsp;canonical&nbsp;{dot}&nbsp;com)</td>
+              </tr>
+
+              <tr>
+                <th scope='row'>Ubuntu Live CD Installer</th>
+                <td>Ubiquity</td>
+                <td>Steve Langasek (steve.langasek&nbsp;{at}&nbsp;canonical&nbsp;{dot}&nbsp;com)</td>
+              </tr>
+
+              <tr>
+                <th scope='row'>Ubuntu OpenStack</th>
+                <td>openstack</td>
+                <td>Ryan Beisner (ryan.beisner&nbsp;{at}&nbsp;canonical&nbsp;{dot}&nbsp;com)</td>
+              </tr>
+
+              <tr>
+                <th scope='row'>Ubuntu One</th>
+                <td>u1db,<br /> ubuntu-sso-client</td>
+                <td>Bret Barker (bret.barker (at) canonical.com</td>
```

Comment:
Email address looks different than the others.

Reply:
fixed and fixed in the sheet